ctest.h

```diff
@@ -486,14 +544,21 @@ int ctest_main(int argc, const char *argv[])
     }
     ctest_end++;    // end after last one
```

Comment:
This leads to the need to subtract 1 twice below. All 3 operations could be safely eliminated.

Reply:
I would argue that they help readability even if they are technically redundant. The way it is currently written, ctest_end always points one past the last entry in the linker set.ctest_end gets decremented only after we've removed the dummy entry with the memmove. What do you think?

Reply:
To me readability is fine either way. I'll let @bvdberg decide :)